src/app/pages/adoption-form/adoption-form.js

```diff
@@ -71,5 +71,4 @@ export default class AdoptionForm extends BaseView {
         this.regions.primaryBook.append(bookInfoView);
         this.toggler.update();
     }
-
```

Comment:
The file still has a blank line at the bottom in my editor, and ends on line 75 (close-brace is on 74). ???